### PR TITLE
feat(dev): Check for Github workflow syntax issues

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/sirosen/check-jsonschema
```
Comment (armenzg) [inline .pre-commit-config.yaml:2]: This is the same pre-commit we use in Sentry

**Comment by untitaker** [comment]
we have no precommit setup at all, or rather just a lightweight, in-house shellscript. is it possible to run this in CI only?

**Comment by armenzg** [comment]
> we have no precommit setup at all, or rather just a lightweight, in-house shellscript. is it possible to run this in CI only?

If we run only on CI, I would not gain any value since Github would show me that the workflow had an invalid syntax.
I'm happy to not go for this since I barely touch the repo. I thought it would be beneficial for whoever has pre-commit already installed on their machines.

Just to be clear, this is optional. Only folks that run `pre-commit install` will get this installed.

**Review by untitaker — APPROVED** [review]

**Comment by untitaker** [comment]
ok just merge it if it's mostly for you

**Pull request merged**